tests/system/providers/qdrant/example_dag_qdrant.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.qdrant.operators.qdrant import QdrantIngestOperator
+
+with DAG(
+    "example_qdrant_ingest",
+    schedule=None,
+    start_date=datetime(2024, 1, 1),
+    catchup=True,
```

Comment:
```suggestion
    catchup=False,
```
Typically we make sure the examples have `catchup=False` just in case users (especially those new to Airflow) change the `schedule` or `start_date`. This way they aren't surprised by spawning DAG runs. Unless this is a deliberate decision (albeit the default)?